ctfpad/models.py

```diff
@@ -463,7 +463,7 @@ def jitsi_url(self):
 
     def save(self):
         if self.flag_tracker.has_changed("flag"):
-            self.status = "solved"
+            self.status = "solved" if self.flag else "unsolved"
```

Comment:
This was meant to go with previous PR sorry. With this we can now un-solve a challenge.